modules/dreamview/backend/teleop/teleop.cc

```diff
@@ -87,14 +98,9 @@ void TeleopService::RegisterMessageHandlers() {
       });
   // Request to get updated modem info for client display
   websocket_->RegisterMessageHandler(
-      "GetModemInfo",
+      "RequestTeleopStatus",
       [this](const Json &json, WebSocketHandler::Connection *conn) {
-        std::string to_send;
-        {
-          boost::shared_lock<boost::shared_mutex> reader_lock(mutex_);
-          MessageToJsonString(modem_info_, &to_send);
```

Comment:
I think we need to preserve this part and save it as a field in teleop_status_ either when receiving modem info or SendStatus.